code/modules/power/supermatter/supermatter.dm

```diff
@@ -440,10 +440,9 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			S.consume(src)
 			return //No boom for me sir
 	else if(power > POWER_PENALTY_THRESHOLD)
-		investigate_log("has spawned additional energy balls.", INVESTIGATE_SUPERMATTER)
+		investigate_log("has released tritium from an overcharge.", INVESTIGATE_SUPERMATTER)
 		if(T)
-			var/obj/singularity/energy_ball/E = new(T)
-			E.energy = power
+			atmos_spawn_air("tritium=2500;TEMP=500000") //extremely fucking hot and could do fusion if you tried hard enough //GEORGE SEARS DID 9/11
```

Comment:
Have you tested this? The engine will just blow up and space all this gas immediately

Reply:
Yeah, i've been thinking on how to make it better.

Reply:
if its an energy based delamination, why not do something like breaking all the lights, causing ion storm and disabling apc's along with the explosion

Reply:
at that point just keep the tesla man, it'll be cheaper.

Reply:
massive emp blast would be cool